doc/api/errors.md

```diff
@@ -2250,6 +2255,17 @@ size.
 This `Error` is thrown when a read is attempted on a TTY `WriteStream`,
 such as `process.stdout.on('data')`.
 
+<a id="ERR_TYPE_MISMATCH"></a>
+#### ERR_TYPE_MISMATCH
```

Comment:
Since we renamed `type` to `entry-type` should this also be renamed accordingly? This seems very broad currently even though it's a pretty specific error.

Reply:
It’s not just a mismatch of `--entry-type` and extension, this also covers a mismatch of `--entry-type` and `package.json` `"type"` field. So it is pretty broad. You could argue that both cases involve `--entry-type`, but naming it `ERR_ENTRY_TYPE_MISMATCH` implies that the problem is on the `--entry-type` side when really it could be on either side. The error message itself specifies what mismatches what.

Reply:
I've renamed it entry type already and force pushed... LOL. We can discuss one last time in today's module meeting... if we need to change it one more time I can. Either way this can easily be updated while we are flagged